### ISSUE TITLE
Issues with Jade

### ISSUE THREAD

**Issue by nco2k** [Kira-NT/nether-chest#10]
when you place a nether chest on the ground and look at it, [Jade](https://github.com/Snownee/Jade) wont show its content. but when you log out and log back in, then the content will show.

response from the Jade dev regarding this [issue](https://github.com/Snownee/Jade/issues/389):
> This should be fixed on their end. Their inventory is not available immediately after the block is placed, which conflicts with Jade's caching strategy

**Comment by Kira-NT** [comment]
Does it update a tooltip whenever a nether chest's channel is changed? Nether chests are rather unusual containers for other mods to deal with.

> Their inventory is not available immediately after the block is placed

Yup, that's true, and this is by design. A nether chest first needs to access the global storage attached to the world its placed in. This usually happens one tick after the block is actually placed.

**Comment by nco2k** [comment]
i never used channels before, but i tried it out and no, it did not update. you have to log out and log back in to see the content.

**Comment by Kira-NT** [comment]
Yeah, I thought so. Nether Chests just don't conform to Jade's expectations on how a container should work. And that's okay, because Nether Chests aren't actually regular containers, as they are not a part of yet another mod that provides 12 retextured versions of the same generic chest. Sometimes mods are not logically compatible. And since here this logical incompatibility doesn't result in a crash, but rather in one missing tooltip, I don't think there's much to be done about that. Making our mods work together perfectly would effectively introduce a special case into the codebase, either on Jade's side or Nether Chest's, which would be too costly to justify for a bunch of hobbyist developers to maintain.

If somebody has time for that and wants to suggest a PR that won't break because of a random Jade update, I'm open to accepting one. So, I'll leave the issue open. However, just like Snownee, I don't see enough value in it to implement this myself.
